contrib/intel/jenkins/build.py

```diff
@@ -0,0 +1,193 @@
+other_bm = args.other_benchmarks
+
+if (args.build_mode):
+    build_mode = args.build_mode
+else:
+    build_mode = 'reg'
+
+def build_libfabric(mode=None):
+
+        if (mode == 'dbg'):
+            config_cmd = ['./configure','--prefix={}'.format(install_path), '--enable-debug',
+                          '--enable-usnic=no', '--enable-psm=no', '--enable-psm2=yes',
+                          '--enable-verbs=yes', '--enable-rxd=yes', '--enable-rxm=yes', 
+                          '--enable-sockets=yes', '--enable-tcp=yes', '--enable-udp=yes',
+                          '--enable-rxd=yes', '--enable-shm=yes', '--with-psm2-src={}/opa-psm2-lib'.format(workspace)]
+        elif(mode == 'dl'):
+            config_cmd = ['./configure','--prefix={}'.format(install_path),
+                          '--enable-usnic=no', '--enable-psm=no', '--enable-psm2=dl',
+                          '--enable-verbs=dl', '--enable-rxd=dl', '--enable-rxm=dl', 
+                          '--enable-sockets=dl', '--enable-tcp=dl', '--enable-udp=dl',
+                          '--enable-rxd=dl', '--enable-shm=dl', '--with-psm2-src={}/opa-psm2-lib'.format(workspace)]
+        else:
+            config_cmd = ['./configure','--prefix={}'.format(install_path),
+                          '--enable-usnic=no', '--enable-psm=no', '--enable-psm2=yes',
+                          '--enable-verbs', '--enable-rxd=yes', '--enable-rxm=yes', 
+                          '--enable-sockets=yes', '--enable-tcp=yes', '--enable-udp=yes',
+                          '--enable-rxd=yes', '--enable-shm=yes', '--with-psm2-src={}/opa-psm2-lib'.format(workspace)]
+
+  
+        if (os.path.exists(install_path) != True):
+            os.makedirs(install_path)  
+    
+        common.run_command(['./autogen.sh'])
+        common.run_command(shlex.split(" ".join(config_cmd)))
+        common.run_command(['make','clean'])
+        common.run_command(['make'])
+        common.run_command(['make','install'])
+
+
+def build_fabtests(mode):
+       
+    os.chdir('{}/fabtests'.format(workspace))
+    if (mode == 'dbg'):   
+        config_cmd = ['./configure', '--enable-debug', '--prefix={}'.format(install_path),
+                '--with-libfabric={}'.format(install_path)] 
+    else:
+        config_cmd = ['./configure', '--prefix={}'.format(install_path),
+                '--with-libfabric={}'.format(install_path)]
+
+
+    common.run_command(['./autogen.sh'])
+    common.run_command(config_cmd)
+    common.run_command(['make','clean'])
+    common.run_command(['make'])
+    common.run_command(['make', 'install'])
+
+
+def build_mpi(mpi, mpisrc, mpi_install_path, build_mode):
+   
+    build_mpi_path ="/mpibuilddir/{}-build-dir/{}/{}/{}".format(mpi, branchname, buildno, build_mode)
+    if (os.path.exists(build_mpi_path) == False):
+        os.makedirs(build_mpi_path)
+    
+    os.chdir(build_mpi_path)
+    if (mpi == 'ompi'):
+        mpistr = '-mpi-'
+    elif (mpi == 'mpich'):
+        mpistr= '-'
```

Comment:
why no string for mpich?

Reply:
the flag for mpich is --enable-fortran while for ompi it is --enable-mpi-fortran